### PR TITLE
added bias to layer, added sigmoid function

### PR DESCRIPTION
**Pull request by kuweg**

### CODE REVIEW

**Comment by kuweg** [comment]
At current dev stage neural network is able to learn using gradient descent algorithm. 
Were created all basic interface methods such as .fit, .predict and some inner logic.
Bugs and further development:
2/ 10 runs are failed due to unknow reason which appers while oprimizing weights and bias -> will be fixed first
Losses will be represented as classes.
Gradient descent will be moved out of model to separate file and will be a new model's parametr.
Some troubles with model's input shape will be fixed, because it's quite uncomfortable to use
